components/metal/CapabilitiesSection.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <section class="capabilities-section">
+    <div class="capabilities-section__container">
+      <div class="capabilities-section__content-wrapper">
+        <CapabilityCard
+          v-for="item in capabilities"
+          :id="item.title"
+          :key="item.title"
+          class="capabilities-section__card"
+          :title="item.title"
+          :description="item.description"
+          :image="item.image"
+        />
+      </div>
+      <div class="capabilities-section__scrolling-ui">
+        <FakeUI class="capabilities-section__scrolling-ui-element">
+          <div
+            v-for="item in capabilities"
+            :key="item.image"
+            class="capabilities-section__image"
+            :class="{
+              'capabilities-section__active-image' : item.title === activeSection
+            }"
+            :lazy-background="item.image"
+          />
+        </FakeUI>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import CapabilityCard from '~/components/metal/CapabilityCard.vue'
+import FakeUI from '~/components/metal/FakeUI.vue'
+import { METAL_CAPABILITIES } from '~/constants/metalContent'
+
+@Component({
+  components: { CapabilityCard, FakeUI }
+})
+export default class extends Vue {
+  capabilities = METAL_CAPABILITIES
+
+  activeSection = ''
+
+  _observer: IntersectionObserver | null = null
+
+  mounted () {
+    const threshold = [...Array(25).keys()].map(x => 4 * x / 100)
+    const windowTriggerMargins = '-16px 0px -66% 0px'
+    this._observer = new IntersectionObserver(
+      this._onSectionAppearing,
+      {
+        root: null, // the viewport
+        rootMargin: windowTriggerMargins,
+        threshold
+      }
+    );
+    (this.$el as HTMLElement)
+      .querySelectorAll('.capabilities-section__card')
+      .forEach((section) => {
+        (this._observer as IntersectionObserver).observe(section)
+      })
+  }
+
+  beforeDestroy () {
+    this._observer && this._observer.disconnect()
+  }
+
+  beforeRouteEnter (route, _, next) {
+    next(overviewPage => overviewPage._parseSectionFromUrl(route))
+  }
+
+  beforeRouteUpdate (route, _, next) {
+    this._parseSectionFromUrl(route)
+    next()
+  }
+
+  _onSectionAppearing (entries: Array<IntersectionObserverEntry>) {
```

Comment:
@korgan00 - in the Figma file, @JRussellHuffman suggested we take a look at the scrolling pattern here: https://www.ibm.com/quantum-computing/network/members/

Russell, can you confirm the pattern here?

In my mind, content on the left will stick (section title and description), as images scroll on the right.

Reply:
As per offline conversation, @JRussellHuffman confirmed the pattern. I consider this resolved